idx/memory/memory.go

```diff
@@ -47,20 +51,26 @@ var (
 	// metric idx.metrics_active is the number of currently known metrics in the index
 	statMetricsActive = stats.NewGauge32("idx.metrics_active")
 
-	Enabled                  bool
-	matchCacheSize           int
-	maxPruneLockTime         = time.Millisecond * 100
-	maxPruneLockTimeStr      string
-	TagSupport               bool
-	TagQueryWorkers          int // number of workers to spin up when evaluation tag expressions
-	indexRulesFile           string
-	IndexRules               conf.IndexRules
+	// Enabled determines if the Memory Index is enabled
+	Enabled             bool
+	matchCacheSize      int
+	maxPruneLockTime    = time.Millisecond * 100
+	maxPruneLockTimeStr string
+	// TagSupport determines if Tag Support is enabled
+	TagSupport bool
+	// TagQueryWorkers is the number of workers to spin up when evaluating tag expressions
+	TagQueryWorkers int
+	indexRulesFile  string
+	// IndexRules holds the path of the Index Rules configuration file
```

Comment:
not true. this comment is probably about the variable above.
most of these comments don't seem to really add any new information so i'm not convinced they're worth the diff noise.